docs/guides/integration-prebid.md

```diff
@@ -1,172 +1,230 @@
 
-UID2 requires initial tokens to be generated server-side. You can do this by calling the [POST /token/generate](../endpoints/post-token-generate.md) endpoint to generate a new UID2 token.
+When you download the Prebid.js package, add the UID2 module by checking the box next to the module named "Unified ID 2.0". The module is listed under the section *User ID Modules*.
 
-## UID2 User ID Submodule
+Once you have added Prebid.js to your site and confirmed that it is working properly, you are ready to configure the UID2 module.
 
-UID2 requires initial tokens to be generated server-side. The UID2 module handles storing, providing, and optionally refreshing them. The module operates in Client Refresh mode.
+:::tip
+Ensure the UID2 module is installed by finding the string "uid2IdSystem" in the [`pbjs.installedModules` array](https://docs.prebid.org/dev-docs/publisher-api-reference/installedModules.html).
+:::
 
->**Important:** UID2 is not designed to be used where GDPR applies. The module checks the passed-in consent data, and does not operate if the `gdprApplies` flag is set to `true`.
+## Configure the UID2 module
 
-## Client Refresh Mode
+To configure the UID2 module, call `pbjs.setConfig` with an object containing the **public key** and **subscription ID** obtained during account setup, as well as the user's [DII](../ref-info/glossary-uid.md#gl-dii) (email address or phone number).
 
-You must provide the Prebid module with the full JSON response body from the applicable endpoint:
+Once configured, the UID2 module will generate a UID2 token for the user and store it in the user's browser. The module will automatically refresh the token as required while your site is open in the user's browser.
 
-- [POST /token/generate](../endpoints/post-token-generate.md) for a new UID2 token.
-- [POST /token/refresh](../endpoints/post-token-refresh.md) for a refreshed UID2 token.
+The user's DII can be passed to the UID2 module hashed or unhashed. If the DII is passed to the UID2 module unhashed, the module will hash it for you. If the DII is passed to the module hashed, it must be normalized before hashing. See [Normalization and Encoding](../getting-started/gs-normalization-encoding.md) for more details.
 
-For an example, see [Sample Token](#sample-token).
+The UID2 module will encrypt the hashed DII before sending it to the UID2 service.
 
-As long as the refresh token remains valid, the module refreshes the UID2 token as needed.
+The following sections demonstrate the different ways to configure the UID2 module and list the requirements for the DII passed to the module.
 
-### Response Storage Options
+:::note
+The examples assume you are using the UID2 production environment. During integration testing use the UID2 integration environment by setting `params.uid2ApiBase` to `"https://operator-integ.uidapi.com"`. You will have a different **subscription ID** and **public key** for the integration environment.
```

Comment:
Agreed with Andrei in that we should not be putting integ data into the bid stream, can we call that out here please?

Reply:
Done

Reply:
Thanks!